From 1218e545f0bce603a5b30241899309a428021ebf Mon Sep 17 00:00:00 2001
From: Jukka Rissanen <jukka.rissanen@linux.intel.com>
Date: Fri, 1 May 2015 12:10:28 +0300
Subject: [PATCH] connman: Enable dependency tracking

The automake option --disable-dependency-tracking does not work nicely
with ConnMan 1.29 release.
The reason is ConnMan commits ac332c5d01b0737c18cb58c8ccc67cf6b0427e1d
and ea6c21b0b20f6f25b25386285d81d859dc843820 that change how and when
various .in files are generated. After those commits, the auto
generated files / directories are created during make. Earlier they
were generated by configure script.
---
 meta/recipes-connectivity/connman/connman.inc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/meta/recipes-connectivity/connman/connman.inc b/meta/recipes-connectivity/connman/connman.inc
index d068ba8..ad1032f 100644
--- a/meta/recipes-connectivity/connman/connman.inc
+++ b/meta/recipes-connectivity/connman/connman.inc
@@ -29,6 +29,7 @@ EXTRA_OECONF += "\
     --enable-test \
     --disable-polkit \
     --enable-client \
+    --enable-dependency-tracking \
 "
 
 PACKAGECONFIG ??= "wispr \
-- 
2.1.0

